app/pages/home.cjsx

```diff
@@ -10,6 +9,11 @@ ProjectCard = require '../partials/project-card'
 alert = require '../lib/alert'
 LoginDialog = require '../partials/login-dialog'
 
+FEATURED_PRODUCT_IDS = if process.env.NODE_ENV is 'production'
```

Comment:
That'd be nice to develop with, but this way envify will turn `process.env.NODE_ENV` into `"production"`, then uglify will turn `"production" === "production"` into `true` and drop the unreachable code in the build. Not a huge optimization in this case, but probably a good habit to set.